ts/src/spl/index.ts

```diff
@@ -0,0 +1,10 @@
+import { Program } from "../index.js";
+import { program as tokenProgram, SplToken } from "./token.js";
+
+export { SplToken } from "./token.js";
+
+export class Spl {
```

Comment:
Might be better to put this into its own package. Presumably tree shaking makes this ok to bundle it in though?